src/components/image-drop-crop/crop.js

```diff
@@ -0,0 +1,42 @@
+import {Observable as $} from 'rx'
+import {div, img} from 'cycle-snabbdom'
+import {dissoc} from 'ramda'
+import Cropper from 'cropperjs'
+require('cropperjs/dist/cropper.css')
+
+const component = ({DOM, props$}) => {
+  const evt$ = DOM.select('.crop').events('crop').share()
+
+  const image$ = evt$.flatMap(evt => {
+    const canvas = evt.originalTarget.cropper.getCroppedCanvas()
```

Comment:
Cycle.js uses `evt.originTarget` not originalTarget, but I did a console.log on this event and it doesn't seem to be patched by cycle-snabbdom, changing it to `evt.target.cropper.getCroppedCanvas()` worked for me locally though.